src/rust/catloop/mod.rs

```diff
@@ -237,6 +250,8 @@ impl CatloopLibOS {
         new_port: u16,
         yielder: Yielder,
     ) -> (QDesc, OperationResult) {
+        #[cfg(feature = "profiler")]
+        timer!("catloop::accept_coroutine");
```

Comment:
I'm not sure if we want these timers since the coroutine might await for a long time. I think we need a more sophisticated way to credit cycles to coroutines.

Reply:
Agree. In d167329a623ef8134656a257ec1c828d0cf9f70b, I removed the annotations of async functions.